src/gevent/pool.py

```diff
@@ -23,11 +23,12 @@
 
 from gevent.hub import GreenletExit, getcurrent, kill as _kill
 from gevent.greenlet import joinall, Greenlet
+from gevent.queue import Full as QueueFull
 from gevent.timeout import Timeout
 from gevent.event import Event
 from gevent.lock import Semaphore, DummySemaphore
 
-__all__ = ['Group', 'Pool']
+__all__ = ['Group', 'Pool', 'Full']
```

Comment:
This causes [a test failure](https://ci.appveyor.com/project/denik/gevent/build/1.0.1025/job/na4vgud16fh8k663#L1079). You'll need to update the name to match.